src/style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+
+  --color1: #060b35;
+  --color2: #616070;
+  --color3: #00acdc;
```

Comment:
For the future: it's better to determine all variables. For instance: `--color-main`, `--color-secondary`, etc. It will have more meaning than just numbers.

Reply:
Ok! Thank you